### PR TITLE
Adding basic card directive

### PR DESCRIPTION
**Pull request by erundle**
Used so that we can create dashboard like cards without specifying the wrapper HTML each time allowing us to focus on the development of the inner card content.

### CODE REVIEW

File: dist/angular-patternfly.js [hunk 1]
```diff
@@ -1,4 +1,12 @@
 /**
+ * @name  patternfly card
+ *
+ * @description
+ *   Card module for patternfly.
+ *
+ */
+angular.module('patternfly.card', []);
+;/**
```
Comment (waldenraines) [inline dist/angular-patternfly.js]: Errant `;`?

File: src/card/basic/card-basic.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div ng-class="showTopBorder === 'true' ? 'card-pf card-pf-accented' : 'card-pf'" class="cardContainer">
```
Comment (waldenraines) [inline src/card/basic/card-basic.html]: What is the purpose of this cardContainer class?  I don't see it used in css anywhere.

Comment (erundle) [inline src/card/basic/card-basic.html]: Yes you are right. It was only for testing purposes but I can access it via card-pf instead

**Comment by jeff-phillips-18** [comment]
LGTM

**Comment by waldenraines** [comment]
Shouldn't the naming of the file "card-basic.directive.js" match the directive name of `pfCard`?

**Comment by erundle** [comment]
Sure renamed to card.directive.js

**Pull request merged**